app/apollo/utils/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2020 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const _ = require('lodash');
+
+const buildSearchForResources = (baseSearch, searchStr = '', fromTime, toTime) => {
+  let ands = [];
+  const tokens = _.filter(searchStr.split(/\s+/));
```

Comment:
Array filter method should work in lieu of lodash

Reply:
fixed, removed logdash dependency in this file.